app/routes/confirmation.js

```diff
@@ -10,9 +10,17 @@ module.exports = {
     try {
       await senders.sendContactDetails(createMsg.getContactDetails(request, confirmationId), request.yar.id)
     } catch (err) {
+      await request.ga.event({
+        category: 'Confirmation',
+        action: 'Error'
+      })
       return h.view('500')
     }
 
+    await request.ga.event({
```

Comment:
put this in the try section, before catch
else it will override the statement in catch

Reply:
that's because this block lines 20 - 23 is executed anyway, whether catch is run or not